app/models/experimentation/treatment_group_membership.rb

```diff
@@ -62,6 +62,8 @@ def record_visit(blood_pressure:, blood_sugar:, prescription_drug:)
         days_to_visit: days_to_visit,
         **visit_status_fields
       )
+    rescue NoMethodError
```

Comment:
Begin/Rescue work when within a blog. You'll need to add a begin block at line 43 in order for the rescue to work.

Reply:
Can't find the documentation for some reason but this works as well. Other places in the code with this style: 
https://github.com/simpledotorg/simple-server/blob/master/app/controllers/admin/facility_groups_controller.rb#L69
https://github.com/simpledotorg/simple-server/blob/master/app/models/reports/drug_stock_calculation.rb#L25